src/pages/trade/model/use-path.ts

```diff
@@ -21,15 +21,12 @@ export const usePathToMetadata = () => {
   const { data, error, isLoading } = useAssets();
   const { baseSymbol, quoteSymbol } = usePathSymbols();
 
-  const baseSymbolNormalized = baseSymbol.toUpperCase();
-  const quoteSymbolNormalized = quoteSymbol.toUpperCase();
-
   const query = useQuery({
     queryKey: ['pathToMetadata', data, baseSymbol, quoteSymbol],
     queryFn: () => {
       return {
-        baseAsset: data?.find(m => m.symbol === baseSymbolNormalized),
-        quoteAsset: data?.find(a => a.symbol === quoteSymbolNormalized),
+        baseAsset: data?.find(m => m.symbol.toLowerCase() === baseSymbol.toLowerCase()),
+        quoteAsset: data?.find(a => a.symbol.toLowerCase() === quoteSymbol.toLowerCase()),
```

Comment:
this fix enabled symbols like `TestUSD` that weren't previously matched by uppercase